### PR TITLE
VSCode extension fails to start on Ubuntu #2738

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <MICGIBSO@uk.ibm.com>

## What type of PR is this ? 

- [ X] Bug fix
- [ ] Enhancement

## What does this PR do ?
Adds a new troubleshooting issue for #2738.

## Which issue(s) does this PR fix ?
https://github.com/eclipse/codewind/issues/2738

## Does this PR require a documentation change ?
Yes.

## Any special notes for your reviewer ?
New content has been Acrolinx checked.

### CODE REVIEW

**Comment by micgibso** [comment]
@hhellyer @BethGriggs Please review, thx.

File: docs/_documentations/troubleshooting.md [hunk 1]
```diff
@@ -1002,6 +1002,20 @@ If you can run `docker` from the command line, but Codewind still fails to find
 3. Open the terminal you ran `docker` from and run `code`.
 4. Now, the new VS Code instance shares the terminal's `PATH`. Start Codewind again.
 
+## VS Code extension fails to start on Ubuntu
+<!--
+Action/Topic: VS Code extension fails to start on Ubuntu
```
Comment (BethGriggs) [inline docs/_documentations/troubleshooting.md:1007]: Part 2 mentions MacOS, should this be shortened to "VSCode extension fails to start"

Comment (micgibso) [reply docs/_documentations/troubleshooting.md]: Good catch, corrected throughout.

File: docs/_documentations/troubleshooting.md [hunk 2]
```diff
@@ -1002,6 +1002,20 @@ If you can run `docker` from the command line, but Codewind still fails to find
...
+Issue type: bug
+Issue link: https://github.com/eclipse/codewind/issues/2738
+Solution: https://github.com/eclipse/codewind/issues/2738#issuecomment-618909203
+Info added in 0.13.0.
+-->
+You install VS Code, it fails to start, and reports a `Version in "/home/ibmuser/.codewind/docker-compose.yaml" is unsupported` error message. The main symptom of this issue is that the `~/codewind/docker-compose.yaml` file is not created when you are installing Codewind. 
+
+**Workaround:**
+1. To fix your Ubuntu VM, update the `docker-compose` version to the latest, and then uninstall and reinstall Codewind.
+2. For macOS, remove the Codewind container that shows when you enter the `$ docker ps -all` command, and then uninstall and reinstall Codewind.
```
Comment (BethGriggs) [inline docs/_documentations/troubleshooting.md:1017]: Ah, I'm now thinking both of these workarounds are not specific to the OS, they're two steps that could be checked on any OS.

> 1. Check your `docker-compose` version is the latest, and then uninstall and reinstall Codewind.
> 2. If there is a Codewind container showing in the output of `$ docker ps -all` , remove the container. Then uninstall and reinstall Codewind. 

What do you think? Other than that this looks good, thank you

Comment (micgibso) [reply docs/_documentations/troubleshooting.md]: Yes, this is better, have made the changes. Thanks.

**Pull request merged**